FirebaseRemoteConfig/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
+- [changed] Add support for other Firebase products to integrate with Remote Config.
```

Comment:
Shouldn't be developer visible, so can be removed.

Reply:
Should we remove the whole entry for Remote Config?

Reply:
Discussed offline, we think it is still worth for developers to know which version we have a major update so we will keep the release note here